core/naive-strategy.go

```diff
@@ -342,6 +344,8 @@ func (st *NaiveStrategy) UnrelayedAcknowledgements(src, dst *ProvableChain, sh S
 		return nil, err
 	}
 
+	defer logger.TimeTrack(now, "UnrelayedAcknowledgements", "num_src", len(srcAcks), "num_dst", len(dstAcks))
```

Comment:
@dongrie Is it intentional to exclude L352-387 from the time tracking?

Reply:
I missed the includeRelayedButUnfinalized part.
https://github.com/hyperledger-labs/yui-relayer/pull/117/commits/9661a8d839c78e4a3ef80c58b4f002653df4dece
Fixed